pkg/project/project.go

```diff
@@ -5,7 +5,7 @@ var (
 	gitSHA             = "n/a"
 	name        string = "aws-operator"
 	source      string = "https://github.com/giantswarm/aws-operator"
-	version            = "8.2.2-dev"
+	version            = "8.2.2"
```

Comment:
This should be changed in a separate commit. See https://gigantic.slack.com/archives/C04TGHDEF/p1583408026036700

Reply:
done